PR: https://github.com/odoo/odoo/pull/

From: 9a7fd72197799c0815f0d2e0ed4f895f674a053d
From: Adrian Torres
Date: 2017-12-12 14:56:59

Breaking data model changes scores: del:7 + add:0, change matches:
-    _inherit = "res.company"
-    social_twitter = fields.Char('Twitter Account')
-    social_facebook = fields.Char('Facebook Account')
-    social_github = fields.Char('GitHub Account')
-    social_linkedin = fields.Char('LinkedIn Account')
-    social_youtube = fields.Char('Youtube Account')
-    social_googleplus = fields.Char('Google+ Account')

Total Changes: 16

[ADD] social_media: new module for social media

This technical dependency module adds settings to res_company for the
company's social media pages.

It is introduced to fix duplicated fields for res_company in website and
mass_mailing, these two modules now require social_media as a dependency

================================= pseudo patch: =================================

--- a/addons/mass_mailing/models/__init__.py
+++ b/addons/mass_mailing/models/__init__.py
@@ -8,4 +8,3 @@ from . import mail_thread
 from . import res_config_settings
 from . import mass_mailing_report
 from . import link_tracker
-from . import res_company

--- a/addons/mass_mailing/models/res_company.py
+++ b/None
@@ -1,15 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import fields, models
-
-class Company(models.Model):
-    _inherit = "res.company"
-
-    # NB: Also defined in website
-    social_twitter = fields.Char('Twitter Account')
-    social_facebook = fields.Char('Facebook Account')
-    social_github = fields.Char('GitHub Account')
-    social_linkedin = fields.Char('LinkedIn Account')
-    social_youtube = fields.Char('Youtube Account')
-    social_googleplus = fields.Char('Google+ Account')
